ydb/mvp/core/mvp_tokens.h

```diff
@@ -34,6 +42,7 @@ class TMvpTokenator : public NActors::TActorBootstrapped<TMvpTokenator> {
             EvRefreshToken = EventSpaceBegin(NActors::TEvents::ES_PRIVATE),
             EvUpdateIamToken,
```

Comment:
to be honest, I'm not sure if it's possible to change the name to EvUpdateIamTokenY here. in this case, the name will better display the selected profile